### ISSUE TITLE
Do I need to set Enable compressed plot support when using the giga36 GUI?

### ISSUE THREAD

**Issue by B1gCarr0t** [madMAx43v3r/chia-gigahorse#327]
![0c4bf4061615367d0e4bdca06b78c4a](https://github.com/madMAx43v3r/chia-gigahorse/assets/85839420/971abbc1-6dd4-48bf-b38f-1f08b2d684b4)

**Comment by spleen911** [comment]
No. That is not needed for GH compressed plots. It can stay disabled if all your compressed plots are GH.

**Comment by B1gCarr0t** [comment]
> No. That is not needed for GH compressed plots. It can stay disabled if all your compressed plots are GH.

谢谢
